lapis2/src/test/kotlin/org/genspectrum/lapis/controller/ExceptionHandlerTest.kt

```diff
@@ -0,0 +1,79 @@
+package org.genspectrum.lapis.controller
+
+import com.ninjasquad.springmockk.MockkBean
+import io.mockk.MockKAnnotations
+import io.mockk.every
+import org.genspectrum.lapis.response.AggregatedResponse
+import org.genspectrum.lapis.silo.SiloException
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest
+import org.springframework.http.MediaType
+import org.springframework.test.web.servlet.MockMvc
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.content
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers.status
+
+@WebMvcTest
+class ExceptionHandlerTest {
+
+    @Autowired
+    lateinit var mockMvc: MockMvc
+
+    @MockkBean
+    lateinit var lapisController: LapisController
+
+    @BeforeEach
+    fun setUp() {
+        MockKAnnotations.init(this)
+    }
+
+    @Test
+    fun `throw NOT_FOUND(404) when route is not found`() {
+        every { lapisController.aggregated(any()) } returns AggregatedResponse(1)
+
+        mockMvc.perform(get("/notAValidRoute"))
+            .andExpect(status().isNotFound)
+    }
+
+    @Test
+    fun `throw INTERNAL_SERVER_ERROR(500) with additional info for any non specific error`() {
+        val validRoute: String = "/aggregated"
+        every { lapisController.aggregated(any()) } throws Exception("SomeMessage")
+
+        mockMvc.perform(get(validRoute))
+            .andExpect(status().isInternalServerError)
+            .andExpect(content().contentType(MediaType.APPLICATION_JSON))
+            .andExpect(
+                content().json(
+                    """
+                    {
+                      "title":"Unexpected error",
+                      "message":"SomeMessage"
+                    }
+                    """.trimIndent(),
```

Comment:
Seems like we don't need the trimIndent?
```suggestion
                    """,
```